wolox-books/src/app/register/register.component.html

```diff
@@ -0,0 +1,34 @@
+<form [formGroup]="rForm" (ngSubmit)="addPost(rForm.value)" class="form">
```

Comment:
Change the name of the form to something more self explanatory. Also, remember to add a `novalidate` to hide all validation warnings made by the HTML.